public/app/features/datasources/settings/BasicSettings.test.tsx

```diff
@@ -11,13 +11,16 @@ const setup = () => {
     onNameChange: jest.fn(),
   };
 
-  return shallow(<BasicSettings {...props} />);
+  return render(<BasicSettings {...props} />);
 };
 
-describe('Render', () => {
-  it('should render component', () => {
-    const wrapper = setup();
+describe('Basic Settings', () => {
+  it('should render component', async () => {
```

Comment:
```suggestion
  it('should render component', () => {
```
This test doesn't need to be async.

Reply:
Thanks I'll change this